homeassistant/components/homewizard/sensor.py

```diff
@@ -425,18 +413,133 @@ class HomeWizardSensorEntityDescription(
     ),
 )
 
+EXTERNAL_SENSORS = {
+    ExternalDevice.DeviceType.GAS_METER: SensorEntityDescription(
+        key="Gas meter",
+        name="Total gas",
```

Comment:
Instead of this, can you add translations?

Reply:
Done 👍